### ISSUE TITLE
Panic: buildssa: package "db"

### ISSUE THREAD

**Issue by cornelk** [golangci/golangci-lint#732]
Version: `golangci-lint has version v1.19.0 built from (unknown, mod sum: "h1:ioXGJ8cIM78dqnHwlfegYLmrsE+N+sU7Vsfy0b6Yhrk=") on (unknown)`

go version: `go version go1.12.9 linux/amd64`

log output:

```
INFO [lintersdb] Active 26 linters: [bodyclose deadcode depguard dogsled errcheck goconst gocritic gocyclo godox gofmt goimports golint gosimple ineffassign maligned misspell nakedret staticcheck structcheck stylecheck typecheck unconvert unparam unused varcheck whitespace] 
INFO [lintersdb] Optimized sublinters [staticcheck gosimple unused stylecheck] into metalinter megacheck 
INFO [loader] Go packages loading at mode 1023 (deps|name|syntax|types_info|compiled_files|exports_file|files|imports|types|types_sizes) took 9.585569159s 
INFO [loader] SSA repr building timing: packages building 47.174149ms, total 1.645289637s 
....
WARN [linters context] Panic: buildssa: package "db" (isInitialPkg: true, needAnalyzeSource: true): interface conversion: *ssa.UnOp is not interface { ssa.setNum(int) }: missing method setNum: goroutine 21305 [running]:
runtime/debug.Stack(0xf457c8, 0x3c, 0xc02cf0d6b0)
	/home/user/go/src/xxx/_tooling/go/current/src/runtime/debug/stack.go:24 +0x9d
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1(0xc05eb99740)
	/home/user/go/src/xxx/vendor/github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:404 +0x1af
panic(0xda0520, 0xc05f17b1a0)
	/home/user/go/src/xxx/_tooling/go/current/src/runtime/panic.go:522 +0x1b5
honnef.co/go/tools/ssa.numberRegisters(0xc05e947180)
	/home/user/go/src/xxx/vendor/honnef.co/go/tools/ssa/func.go:278 +0x114
honnef.co/go/tools/ssa.(*Function).finishBody(0xc05e947180)
	/home/user/go/src/xxx/vendor/honnef.co/go/tools/ssa/func.go:410 +0x746
honnef.co/go/tools/ssa.(*Package).build(0xc05f143920)
	/home/user/go/src/xxx/vendor/honnef.co/go/tools/ssa/builder.go:2352 +0x94e
sync.(*Once).Do(0xc05f14394c, 0xc004bfbc70)
	/home/user/go/src/xxx/_tooling/go/current/src/sync/once.go:44 +0xb3
honnef.co/go/tools/ssa.(*Package).Build(0xc05f143920)
	/home/user/go/src/xxx/vendor/honnef.co/go/tools/ssa/builder.go:2260 +0x54
honnef.co/go/tools/internal/passes/buildssa.run(0xc05f1875e0, 0x2abf99cc9, 0x18b43a0, 0x2, 0x2)
	/home/user/go/src/xxx/vendor/honnef.co/go/tools/internal/passes/buildssa/buildssa.go:70 +0x27b
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc05eb99740)
	/home/user/go/src/xxx/vendor/github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:495 +0x85b
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc05eb99740)
	/home/user/go/src/xxx/vendor/github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:407 +0x51
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func3(0xc05eeed9e0, 0xc05eb99740)
	/home/user/go/src/xxx/vendor/github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:961 +0x5f
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
	/home/user/go/src/xxx/vendor/github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:956 +0x2cd 
```

**Comment by jirfag** [comment]
hi, how did you build golangci-lint? I guess you didn't use binary releases and didn't use replacements from `go.mod`. And because of missing https://github.com/golang/tools/pull/162 you are getting panic

**Comment by sebrandon1** [comment]
I'm hitting this same thing.  I'm following the README (replacing 1.18.0 with 1.19.1):

`GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.19.1`

And I'm getting the same failure when linting my project.

`WARN [linters context] Panic: buildssa: package "server" (isInitialPkg: true, needAnalyzeSource: true): interface conversion: *ssa.FieldAddr is not interface { ssa.setNum(int) }: missing method setNum: goroutine 13769 [running]:`

**Comment by jirfag** [comment]
Just use binary releases

**Comment by sebrandon1** [comment]
I'm using Go 1.12.10.  I got it working using a binary release of v1.19.1.

**Comment by jirfag** [comment]
yep, the bug was fixed only in Go 1.13
